extensions/positron-python/python_files/posit/pyproject.toml

```
@@ -26,6 +26,8 @@ dependencies = [
     "polars",
     "polars[timezone] ; sys_platform == 'win32'",
     "pyarrow",
+    "pymssql ; python_full_version < '3.14'",
+    "pyodbc",
```

It looks like this is marked > 3.14 in the `uv.lock` file

```suggestion
    "pyodbc; python_full_version < '3.14'",
```